Source/gui/SNTBinaryMessageWindowView.swift

```diff
@@ -312,11 +341,50 @@ struct SNTBinaryMessageWindowView: View {
       }
     }
 
-    let url = SNTBlockMessage.eventDetailURL(for: event, customURL: customURL as String?)
+    if let callback = replyCallback {
+      callback(false)
+    }
+
     window?.close()
-    if let url = url {
+
+    let detailsURL = SNTBlockMessage.eventDetailURL(for: event, customURL: customURL as String?)
+
+    if let url = detailsURL {
       openURL(url)
     }
+
+  }
+
+  // This button is only shown when the standalone mode is enabled in place of
+  // the "Open Event" button.
+  func standAloneButton() {
+
+    var msg = "Allow execution of "
+
```

Comment:
```suggestion
    var msg = "allow execution of "

```

Reply:
You might actually have to copy this into the NSLocalizedString on lines 371 and 373, otherwise the localization tools won't pick it up properly